File: SDKs/amazonaws.com/lambda/2015-03-31/python/src/sdk/models/shared/filesystemconfig.py
import dataclasses
from dataclasses_json import dataclass_json
from sdk import utils


@dataclass_json
@dataclasses.dataclass
class FileSystemConfig:
    r"""FileSystemConfig
    Details about the connection between a Lambda function and an <a href=\"https://docs.aws.amazon.com/lambda/latest/dg/configuration-filesystem.html\">Amazon EFS file system</a>.
    """
    
    arn: str = dataclasses.field(metadata={'dataclasses_json': { 'letter_case': utils.field_name('Arn') }})
    local_mount_path: str = dataclasses.field(metadata={'dataclasses_json': { 'letter_case': utils.field_name('LocalMountPath') }})
    

File: SDKs/amazonaws.com/lambda/2015-03-31/java/lib/src/main/java/openapisdk/models/shared/FileSystemConfig.java
package openapisdk.models.shared;

import com.fasterxml.jackson.annotation.JsonProperty;

/**
 * FileSystemConfig
 * Details about the connection between a Lambda function and an <a href="https://docs.aws.amazon.com/lambda/latest/dg/configuration-filesystem.html">Amazon EFS file system</a>.
**/
public class FileSystemConfig {
    @JsonProperty("Arn")
    public String arn;
    public FileSystemConfig withArn(String arn) {
        this.arn = arn;
        return this;
    }
    @JsonProperty("LocalMountPath")
    public String localMountPath;
    public FileSystemConfig withLocalMountPath(String localMountPath) {
        this.localMountPath = localMountPath;
        return this;
    }
}